python/pyspark/ml/tests.py

```diff
@@ -145,6 +145,48 @@ def test_pipeline(self):
         self.assertEqual(5, transformer3.dataset_index)
         self.assertEqual(6, dataset.index)
 
+    def test_pipeline_presistency(self):
```

Comment:
typo: "persistence"

Also, can you please put this test with the other Persistence tests in this file?